specification/network/resource-manager/Microsoft.Network/stable/2019-09-01/expressRouteGateway.json

```diff
@@ -598,6 +598,10 @@
         "routingWeight": {
           "type": "integer",
           "description": "The routing weight associated to the connection."
+        },
+        "enableInternetSecurity": {
```

Comment:
Can this be modeled as an Enum? We try to stay away from boolean flags as they aren't forward compatible and can cause confusion when multiple dependent flags are introduced.

The field could be called either 'InternetSecurity' with values of 'enabled' / 'disabled' (you could easily add different modes in the future this way. Or you could call it something like 'connectionFeatures' with current value limited to 'InternetSecurity'.

Reply:
There are other resources (vpn connection, hubvirtualnetworkconnection) which have the same boolean flag, and for now we would want to keep it the same to maintain consistency.